gwctl/.goreleaser.yaml

```diff
@@ -0,0 +1,35 @@
+# https://goreleaser.com/
+#
+# For local testing, use:
+# 
+#   goreleaser release --snapshot --clean
+project_name: gwctl
+
+before:
+  hooks:
+    # Ensure the project builds successfully before release.
+    - go mod tidy
+    - go build -o ./dist/gwctl ./main.go
```

Comment:
At some point, both this and the repo CI should probably include a test suite. Much larger change, non-blocking for this one, but something to put on your todo.

Reply:
Absolutely yes.